tests/test_measures_sparse.py

```diff
@@ -35,41 +35,17 @@ class TestSparseExpval:
     def dev(self, request):
         return qml.device("lightning.qubit", wires=2, c_dtype=request.param)
 
-    @pytest.mark.skipif(
-        Kokkos_info()["USE_KOKKOS"] == True, reason="Kokkos and Kokkos Kernels are present."
-    )
-    def test_create_device_with_unsupported_dtype(self, dev):
-        @qml.qnode(dev, diff_method="parameter-shift")
-        def circuit():
-            qml.RX(0.4, wires=[0])
-            qml.RY(-0.2, wires=[1])
-            return qml.expval(
-                qml.SparseHamiltonian(
-                    qml.Hamiltonian([1], [qml.PauliX(0) @ qml.Identity(1)]).sparse_matrix(),
-                    wires=[0, 1],
-                )
-            )
-
-        with pytest.raises(
-            NotImplementedError,
-            match="The expval of a SparseHamiltonian requires Kokkos and Kokkos Kernels.",
-        ):
-            circuit()
-
     @pytest.mark.parametrize(
         "cases",
         [
-            [qml.PauliX(0) @ qml.Identity(1), 0.00000000000000000],
-            [qml.Identity(0) @ qml.PauliX(1), -0.19866933079506122],
-            [qml.PauliY(0) @ qml.Identity(1), -0.38941834230865050],
-            [qml.Identity(0) @ qml.PauliY(1), 0.00000000000000000],
-            [qml.PauliZ(0) @ qml.Identity(1), 0.92106099400288520],
-            [qml.Identity(0) @ qml.PauliZ(1), 0.98006657784124170],
+            [qml.PauliX(0) @ qml.Identity(1), 0.00000000000000000, 1.000000000000000000],
+            [qml.Identity(0) @ qml.PauliX(1), -0.19866933079506122, 0.960530638694763184],
+            [qml.PauliY(0) @ qml.Identity(1), -0.38941834230865050, 0.848353326320648193],
+            [qml.Identity(0) @ qml.PauliY(1), 0.00000000000000000, 1.000000119209289551],
+            [qml.PauliZ(0) @ qml.Identity(1), 0.92106099400288520, 0.151646673679351807],
+            [qml.Identity(0) @ qml.PauliZ(1), 0.98006657784124170, 0.039469480514526367],
```

Comment:
Still wondering about the old values here. Maybe we should double check this analytically.

Reply:
We could, but the numbers matched when directly inputting say `qml.PauliY(0) @ qml.Identity(1)` (no `Hamiltonian`, `SparseHamiltonian`). I can check it again though. To be sure, by "old values" you mean the second (expval) column, right?

Reply:
Yep, that's right. Fine by me with it being defined anyway that seems sensible.